pkg/booleanpolicy/validate.go

```diff
@@ -58,6 +58,17 @@ func Validate(p *storage.Policy, options ...ValidateOption) error {
 	for _, section := range p.GetPolicySections() {
 		errorList.AddError(validatePolicySection(section, configuration, p.GetEventSource()))
 	}
+
+	// Special case for ImageSignatureVerifiedBy policy for which we don't allow
```

Comment:
Add forbid operator clause to field metadata registration.  That way the need for explicit validation is removed.

Reply:
Forbid operator in metadata would forbid both AND and OR.

Reply:
Got it. Please check the new commit.